config/drupal-10/drupal-10.0-deprecations.php

```diff
@@ -9,6 +9,12 @@
 use Rector\Symfony\Set\TwigSetList;
 
 return static function (RectorConfig $rectorConfig): void {
+    if (defined('TwigSetList::TWIG_24')) {
+        $twigSet = TwigSetList::TWIG_24;
+    } else {
+        $twigSet = TwigSetList::TWIG_240;
+    }
```

Comment:
Also `use Rector\Exception\ShouldNotHappenException;` should be added.
```suggestion
if (defined(TwigSetList::class .'::TWIG_24')) {
    $twigSet = TwigSetList::TWIG_24;
  } elseif (defined(TwigSetList::class .'::TWIG_240'))  {
    $twigSet = TwigSetList::TWIG_240;
  } else {
    throw new ShouldNotHappenException('Could not detect twig set.');
  }
```